scripts/microbe_stage/microbe_stage_hud.as

```diff
@@ -106,17 +106,21 @@ class MicrobeStageHudSystem : ScriptSystem{
         if(player != NULL_OBJECT){
 
             auto bag = World.GetComponent_CompoundBagComponent(player);
+			auto localPlayer = MicrobeOperations::getSpeciesComponent(World, "Default");
```

Comment:
`localPlayer` should also be renambed here. There's also a bunch of tabs in this block of code.